packages/common/utilities/__tests__/warning.test.ts

```diff
@@ -0,0 +1,37 @@
+// we are testing Console function
+/* eslint-disable no-console */
```

Comment:
Again, reasons should follow the disabling statement so they are better connected to it.

https://eslint.org/docs/latest/use/configure/rules#comment-descriptions

Reply:
Fixed.